prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migration_toolkit/CreateApplicationTask.java

```diff
@@ -0,0 +1,88 @@
+package com.redhat.parodos.tasks.migration_toolkit;
+
+import com.redhat.parodos.workflow.exception.MissingParameterException;
+import com.redhat.parodos.workflow.parameter.WorkParameter;
+import com.redhat.parodos.workflow.parameter.WorkParameterType;
+import com.redhat.parodos.workflow.task.BaseWorkFlowTask;
+import com.redhat.parodos.workflows.work.DefaultWorkReport;
+import com.redhat.parodos.workflows.work.WorkContext;
+import com.redhat.parodos.workflows.work.WorkReport;
+import com.redhat.parodos.workflows.work.WorkStatus;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+
+import java.net.URI;
+import java.util.List;
+
+import static com.redhat.parodos.workflow.context.WorkContextDelegate.getOptionalValueFromRequestParams;
+import static com.redhat.parodos.workflow.context.WorkContextDelegate.getRequiredValueFromRequestParams;
+
+/**
+ * {@link CreateApplicationTask} Creates an application under the application inventory
+ * with the details of the source repository to scan.
+ */
+@Slf4j
+public class CreateApplicationTask extends BaseWorkFlowTask {
```

Comment:
it's suggested to extend from BaseInfrastructureWorkFlowTask. but not too big deal

Reply:
Ack